### PR TITLE
T5-267 partner page

### PR DESCRIPTION
**Pull request by elite0226**

### CODE REVIEW

File: src/components/Menu/Menu.js [hunk 1]
```diff
@@ -363,21 +363,6 @@ const Menu = ({ menuOpen, toggleOpen }) => {
               Gatsby
             </Link>
           </div>
-          <div>
```
Comment (naeluh) [inline src/components/Menu/Menu.js:366]: If we remove this from the nav the only way to get to big commerce is through the footer link Is this correct ?

Comment (elite0226) [reply src/components/Menu/Menu.js:366]: Big commerce removal is required in Jira ticket and we can go to the Big Commerce partner page in Partners page 
`/partners`

Comment (naeluh) [reply src/components/Menu/Menu.js:366]: Ok I can't find the ticket in Jira can you link I just want to double check that this meets all the requirements. Thanks!

File: src/pages/partners/index.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+      src: BigcommerceBadge,
+      alt: 'bigcommerce',
+      label: 'Big Commerce',
+      link: '/partners/big-commerce',
+    },
+  ];
+
+  const handleGoTo = link => () => {
+    navigate(link);
+  };
```
Comment (naeluh) [inline src/pages/partners/index.js:51]: could this just be ?
```js
const handleGoTo = link => navigate(link);
```

File: src/components/Menu/Menu.js (hunk 1 above)
Comment (elite0226) [reply src/components/Menu/Menu.js:366]: https://thirdandgrove.atlassian.net/browse/T5-267
The first requirement is taking the BigCommerce off form the partners list in the menu pulldown.

**Review by naeluh — APPROVED** [review]

**Pull request merged**